homeassistant/components/sensor/systemmonitor.py

```diff
@@ -40,7 +40,7 @@
     'packets_in': ['Packets in', ' ', 'mdi:server-network'],
     'packets_out': ['Packets out', ' ', 'mdi:server-network'],
     'process': ['Process', ' ', 'mdi:memory'],
-    'processor_use': ['Processor use', '%', 'mdi:memory'],
+    'processor_use_percent': ['Processor use (percent)', '%', 'mdi:memory'],
```

Comment:
The processor usage is only exposed as one value unlike the ones for memory and disk usage.

This is a breaking change for something that can be changed by the users already.